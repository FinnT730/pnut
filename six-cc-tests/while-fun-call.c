int main() {
  int_ptr f1;
  int_ptr f2;
  char c1;
  char c2;
  f1 = fopen("six-cc-tests/fgetc.c", 0);
  f2 = fopen("six-cc-tests/while-fun-call.c", 0);
  while ((c1 = fgetc(f1)) && (c2 = fgetc(f2)) && c1 != EOF && c2 != EOF) {
    putchar(c1);
    putchar(c2);
  }
}
