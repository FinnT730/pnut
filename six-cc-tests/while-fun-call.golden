iinntt  mmaaiinn(())  {{

    FiInLtE__ppttrr  ff1;;

    cihnatr_ pct;r
  f 2f; 
=   fcohpaern (c"1s;i
x - ccch-atre sct2s;/
f g eft1c .=c "f,o p0e)n;(
" s iwxh-iclce- t(e(sct s=/ ffggeettcc.(cf"),)  0!)=; 
E O Ff)2  {=
  f o p epnu(t"cshiaxr-(ccc)-;t
e s t}s
/}w
h